jenkins/scripts/select-compiler.sh

```diff
@@ -99,12 +99,10 @@ elif [ "$SELECT_ARCH" = "AIXPPC" ]; then
   case $NODE_NAME in
     *aix72* )
       echo "Setting compiler for Node version $NODEJS_MAJOR_VERSION on AIX7.2"
-
       if [ "$NODEJS_MAJOR_VERSION" -gt "9" ]; then
-        export LIBPATH=/opt/gcc-6.3/lib/gcc/powerpc-ibm-aix7.2.0.0/6.3.0/pthread/ppc64:/opt/gcc-6.3/lib
-        export PATH="/opt/ccache-3.7.4/libexec:/opt/gcc-6.3/bin:/opt/freeware/bin:$PATH"
+        export PATH="/opt/ccache-3.7.4/libexec:/opt/freeware/bin:$PATH"
         export CC="gcc" CXX="g++" CXX_host="g++"
-        echo "Compiler set to 6.3"
+        echo "Compiler set to 8.3"
```

Comment:
Is this intended? It means that if we start using 7.2 for older builds it will use the newer compiler level which I'm not sure would be intended.

Reply:
yes it is turns out the 7.2 machines only have 8.3 installed and have been building with it since I switched them on. However saying that im not sure if building with 8.3 would break people trying to run with 6.3? 

Thoughts @richardlau ?

Reply:
We've already established that binaries built on AIX 7.2 don't run on AIX 7.1 (https://github.com/nodejs/build/issues/2547#issuecomment-782107370) -- is not building with gcc 6 the issue there?

I've always been a bit hesitant building with gcc 6 on AIX 7.2 as we have to go out of our way to get it on there as the default gcc installation on AIX 7.2 via the AIX toolbox is 8.

Reply:
the 7.2 vs 7.1 is not GCC related thats an pure AIX issue, but if the older lines say we build with 6.3 on 7.2 I reckon we should at least test they run on a system with GCC6 when built with GCC8 - I cant see why they wouldnt work

Reply:
turns out gcc6 is installed so ive added back the check for gcc6